packages/core-bridge/scripts/build.js

```diff
@@ -24,6 +24,13 @@ const platformMapping = { darwin: 'apple-darwin', linux: 'unknown-linux-gnu', wi
 
 function compile(target) {
   console.log('Compiling bridge', { target });
+  // Extract version from meta package for use as version number in client
+  // version field.
+  const metaPkg = require('../../meta/package.json');
+  const cargoOutputDir = path.resolve(__dirname, '..', 'target');
+  fs.mkdirSync(cargoOutputDir, { recursive: true });
+  fs.writeFileSync(path.resolve(cargoOutputDir, 'metaversion'), metaPkg.version);
```

Comment:
I'd rather avoid writing to the FS for this, instead just pass it in at runtime.
We can use the worker version for now, at some point I want to sync the versions of the different packages: #244

Reply:
This can cause problems later on if we build before bumping the version in our publish process.